### DISCUSSION TITLE
如何平滑升级哦？

### DISCUSSION THREAD

**Discussion by TeddyXiong79** [IrineSistiana/mosdns#758] [Q&A]
当前环境是iStoreOS的软路由，已经安装好了MosDNS 5.1.3-10版本，请问该如何平滑升级到最新版本5.3.1哦？
是不是直接运行安装脚本就可以？
sh -c "$(curl -ksS https://raw.githubusercontent.com/sbwml/luci-app-mosdns/v5/install.sh)"

**Comment by TeddyXiong79** [comment]
大家都没有升级需求吗？？？？

**Comment by Weiradio1128** [comment]
這篇太有幫助了
